README.md

```diff
@@ -1,16 +1,107 @@
+
+### Bravery
+
+Solving problems that no one has completed before, or that have stymied the most
+brilliant humans ever to walk this earth is daunting. Who are _you_, who am _I_,
+who are _we_ to dare such things?
+
+The human kind.
+
+The kind that descended the trees, harnessed fire, built civilizations in river
+valleys and put our footprint on the Moon. All of these were hard problems that
+all the geniuses and sages _before_ couldn't solve, until someone dared to try.
+Embrace uncertainty, but also embrace your inner warrior. You are in a battle
+against your own unknowing. You can win.
+
+### Focus
+
+We mentioned it under another name as "mindfulness" but while "mindfulness"
+recalls yoga poses and self-help books, it has a practical side we call
+"focus." When you feel the distraction machines (phones, notification pings, a
+morbid curiosity about whatever happened to nobody you really care about on
+some reality show that society would be better off without) clawing at your
+attention, realize that they are multi-_billion_ dollar industries that profit
+from keeping your attention like an addict and selling it off by the half-hour.
+
+The only thing keeping you from this fate is ***your will*** &mdash; which they
+launch focused campaigns to weaken.
+
+In general, the idea is "don't get up until it's done." Now obviously this can
+be a bad thing, sometimes you need food, sometimes a 10 minute walk will help
+you think of a solution you might not have seen previously. But in general,
+when you hit a bug, when you hit a problem, don't get up from the computer.
+Stick with the problem until it's done. Don't "lose" your game face.
+
+Another practice that will help you here is, before you start building, to
+break down each component of the solution into a "deliverable." Make promises
+to yourself that you'll go to dinner, you'll make that phone call, you'll waste
+15 minutes on social media once that deliverable is working, tested, clean, and
+saved.
+
+## Don't Flail
+
+Work from a problem solving process. Don't randomly add code and hope that this
+next run will somehow magically get the answer. Make sure that each time you
+run the code that you expect it to output some validation of your direction
+toward the answer or the answer itself.
+
+## Don't Curate the Internet's Solutions
+
+Many of the challenges we give at FIS are standard problems that require very
+little external pre-existing knowledge. As a result most programming languages
+have built-in or readily copy-and-pasteable solutions. But **_this is not
+programming_**. Learn from the solutions, see the part you're missing, update
+your process. But if you simply use someone else's solution but don't enrich
+your mental models you've made the poorest investment possible.
+
+## Respect the Problem
+
+Lastly, and perhaps the most important, guideline: respect the problem.
+Sometimes the problem seems simple, but when you delve into it's actually very
```

Comment:
Sometimes a problem seems simple, but when you delve into it, it's actually very